src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -311,7 +331,7 @@ public bool CheckDataSourceExists(string dataSourceName)
     public string GetDefaultDataSourceName()
 #pragma warning restore CA1024 // Use properties where appropriate
     {
-        return _defaultDataSourceName;
+        return DefaultDataSourceName;
```

Comment:
This function comment clearly says they preferred to use private field. instead of making it a public field, can we create a public property since you need not restrict the set?

If you do that, this function is no longer useful. We can get rid of it and the comments. 

When making changes to an existing function, always read the context surrounding it and understand how best to integrate your changes into the function such that it is still meaningful. If its no longer meaningful, feel empowered to make the necessary changes that make it meaningful.

Reply:
because we need to update the DefaultDataSourceName from outside of the Runtimeconfig, I think it must be made public. I will remove this function.